INSTALL.md

```diff
@@ -56,11 +56,13 @@ Example: In the case of stock eZ Platform 1.11 and higher that would specificall
 ```
 
 
-### Enable EzPublishLegacyBundle
+### Enable EzPublishLegacyBundle and optionally EzSystemsEzPlatformXmlTextFieldTypeBundle
 Edit `app/AppKernel.php`, and add `new eZ\Bundle\EzPublishLegacyBundle\EzPublishLegacyBundle( $this ),`
 at the end of  the `$bundles` array. Pay close attention to the `$this` argument, LegacyBundle is a bit 
 spoiled and has high expectations from its collaborators ;)
 
+If you need support for legacy `ezxmltext` field type in eZ Publish kernel, enable `EzSystems\EzPlatformXmlTextFieldTypeBundle\EzSystemsEzPlatformXmlTextFieldTypeBundle` bundle.
```

Comment:
maybe something like:
`If you still plan to have content in XmlText _(as opposed to migrating to RichText straight away)_, then you'll also need to enable ...`

Reply:
the updated version is +1 as well